draft-irtf-cfrg-vdaf.md

```diff
@@ -754,11 +754,14 @@ Each VDAF is identified by a unique, 32-bit integer `ID`. Identifiers for each
 
 ## Sharding {#sec-vdaf-shard}
 
-Sharding is syntactically identical to DAFs (cf. {{sec-daf-shard}}):
-
-* `Vdaf.measurement_to_input_shares(measurement: Measurement) -> (Bytes,
-  Vec[Bytes])` is the randomized input-distribution algorithm run by each
-  Client. It consumes the measurement and produces a public share, distributed
+Sharding transforms a measurement into input shares as it does in DAFs 
+(cf. {{sec-daf-shard}}); in VDAFs it additionally takes a nonce as input and 
+produces a public share:
+
+* `Vdaf.measurement_to_input_shares(measurement: Measurement, nonce: Bytes) -> 
+  (Bytes, Vec[Bytes])` is the randomized input-distribution algorithm run by each
+  Client. It consumes the measurement and the nonce and 
```

Comment:
* `make` fails if a line ends with trailing white space:
  ```
  $ make
  757:Sharding transforms a measurement into input shares as it does in DAFs
  758:(cf. {{sec-daf-shard}}); in VDAFs it additionally takes a nonce as input and
  761:* `Vdaf.measurement_to_input_shares(measurement: Measurement, nonce: Bytes) ->
  763:  Client. It consumes the measurement and the nonce and
  draft-irtf-cfrg-vdaf.md contains trailing whitespace
  *** Run 'make fix-lint' to automatically fix some errors
  make: *** [lint-whitespace] Error 1
  ```

* Also, lines should break at 80 characters.

```suggestion
  Client. It consumes the measurement and the nonce and
```